monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -2,34 +2,65 @@ import React from 'react';
 import {BrowserRouter as Router, Redirect, Route, Switch} from 'react-router-dom';
 import {Container} from 'react-bootstrap';
 
-import GettingStartedPage from 'components/pages/GettingStartedPage';
-import ConfigurePage from 'components/pages/ConfigurePage';
-import RunMonkeyPage from 'components/pages/RunMonkeyPage/RunMonkeyPage';
-import MapPage from 'components/pages/MapPage';
-import TelemetryPage from 'components/pages/TelemetryPage';
-import StartOverPage from 'components/pages/StartOverPage';
-import ReportPage from 'components/pages/ReportPage';
-import LicensePage from 'components/pages/LicensePage';
-import AuthComponent from 'components/AuthComponent';
-import LoginPageComponent from 'components/pages/LoginPage';
-import RegisterPageComponent from 'components/pages/RegisterPage';
+import ConfigurePage from './pages/ConfigurePage.js';
+import RunMonkeyPage from './pages/RunMonkeyPage/RunMonkeyPage';
+import MapPage from './pages/MapPage';
+import TelemetryPage from './pages/TelemetryPage';
+import StartOverPage from './pages/StartOverPage';
+import ReportPage from './pages/ReportPage';
+import LicensePage from './pages/LicensePage';
+import AuthComponent from './AuthComponent';
+import LoginPageComponent from './pages/LoginPage';
+import RegisterPageComponent from './pages/RegisterPage';
+import LandingPage from "./pages/LandingPage";
 import Notifier from 'react-desktop-notification';
-import NotFoundPage from 'components/pages/NotFoundPage';
+import NotFoundPage from './pages/NotFoundPage';
+import GettingStartedPage from './pages/GettingStartedPage';
 
 
 import 'normalize.css/normalize.css';
 import 'react-data-components/css/table-twbs.css';
 import 'styles/App.css';
 import 'react-toggle/style.css';
 import 'react-table/react-table.css';
-import notificationIcon from '../images/notification-logo-512x512.png';
 import {StandardLayoutComponent} from './layouts/StandardLayoutComponent';
 import LoadingScreen from './ui-components/LoadingScreen';
+import {DisabledSidebarLayoutComponent} from "./layouts/DisabledSidebarLayoutComponent";
+import {CompletedSteps} from "./side-menu/CompletedSteps";
+import Timeout = NodeJS.Timeout;
+import IslandHttpClient from "./IslandHttpClient";
 
-const reportZeroTrustRoute = '/report/zeroTrust';
-const islandModeRoute = '/api/island-mode'
+
+let notificationIcon = require('../images/notification-logo-512x512.png');
+
+const Routes = {
+  LandingPage: '/landing-page',
+  GettingStartedPage: '/',
```

Comment:
Shouldn't the landing page be `/` and the getting started page be `/getting-started`?

Reply:
The `/landing-page` is only default as long as the mode is not chosen